common/startup/check_volumes_exist_ownership.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function check_volumes_exist_ownership {
+    local VOLUME_ERROR_FOUND=0
+  for volume in ${MAUTIC_VOLUMES}; do
+    if [[ ! -d "${volume}" ]]; then
+      echo "Error: ${volume} does not exist or is not a directory."
+      VOLUME_ERROR_FOUND=1
+    fi
+  done
+
+  if [[ $VOLUME_ERROR_FOUND -eq 1 ]]; then
+    echo "----- Startup Checks Error Found -----" >&2
+    echo "Please ensure the volume(s) are mounted correctly and is accessible." >&2
+    echo "If you are running this container as a non-root user, you may need to run the container with elevated privileges." >&2
+    exit 1
+  fi
+
+  PERM_ERROR_FOUND=0
```

Comment:
```suggestion
    local PERM_ERROR_FOUND=0
```

Shouldn't it be local?